src/components/InstallTabs/index.tsx

```diff
@@ -56,28 +55,32 @@ const getOSPanel = (userOS: UserOS): JSX.Element => {
   }
 };
 
-const InstallTabs = (): JSX.Element | null => {
-  const { userOS } = useDetectOs();
+const os = {
+  win: 'Windows (Chocolatey)',
+  mac: 'macOS (nvm)',
+  linux: 'Linux (nvm)',
+};
 
-  const os = {
-    win: 'Windows (Chocolatey)',
-    mac: 'macOS (nvm)',
-    linux: 'Linux (nvm)',
-  };
+const installTabSystems: Record<UserOS, Array<string>> = {
```

Comment:
Any reason you went with `Array<string>` instead of `string[]`?

Reply:
None, is there a common practice here for not using `Array<T>`? (I'm fine either ways)

Reply:
no just curious. I think most of our code uses string[]